src/docs/product/relay/getting-started.mdx

```diff
@@ -167,17 +182,19 @@ Under default configuration, Relay emits log output similar to:
 ```
  INFO  relay::setup > launching relay from config folder .relay
  INFO  relay::setup >   relay mode: managed
- INFO  relay::setup >   relay id: f2119bc9-9a9b-4531-826b-24e9794902f2
+ INFO  relay::setup >   relay id: cde0d72e-0c4e-4550-a934-c1867d8a177c
  INFO  relay::setup >   public key: QPBITKKtKUuEZGGbPke8iufEXAcVrEv6nmWrkRtc3l8
```

Comment:
Wanna update the public key to `nDJI79SbEYH9-8NEJAI7ezrgYfoIPW3Bnkg00k1z0fA`, too?